addons/website/static/src/interactions/animation.js

```diff
@@ -56,7 +56,7 @@ export class Animation extends Interaction {
 
     setup() {
         this.wrapwrapEl = document.querySelector("#wrapwrap");
-        this.windowUnlessDropdown = this.el.closest(".dropdown") ? [] : window;
+        this.windowUnlessDropdown = this.el.closest(".dropdown") ? null : window;
```

Comment:
What about fixing it globally instead ?
```diff
+++ b/addons/web/static/src/public/colibri.js
@@ -210,7 +212,7 @@ export class Colibri {
         if (sel in selectors) {
             const elem = selectors[sel]();
-            return elem ? [elem] : [];
+            return elem ? (Symbol.iterator in elem ? elem : [elem]) : [];
         }
```
This would also make it possible to associate a set of nodes like for normal selectors - not supporting it was accidental.